### PR TITLE
Document non-public cloud support for Azure Data Explorer scaler.

### PR DESCRIPTION
**Pull request by v-shenoy**
Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

Update documentation for Azure Data Explorer scaler.
https://github.com/kedacore/keda/issues/2734

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.7/scalers/azure-data-explorer.md [hunk 1]
```diff
@@ -40,6 +44,7 @@ triggers:
 - `tenantId` - Id of the Azure AD tenant.
 - `clientId` - Id of the Azure AD application.
 - `clientSecret` - Password of the Azure AD application.
+- `activeDirectoryEndpoint` - Active Directory endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
```
Comment (tomkerkhove) [inline content/docs/2.7/scalers/azure-data-explorer.md:47]: Can you document the cloud configuration option here please?

Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-data-explorer.md:47]: Whoops, bad oversight by me.

Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-data-explorer.md:48]: Done.

Comment (tomkerkhove) [reply content/docs/2.7/scalers/azure-data-explorer.md:48]: That's what reviews are for, thanks!

**Review by tomkerkhove — APPROVED** [review]

**Review by JorTurFer — APPROVED** [review]
LGTM

**Pull request merged**